masterlist.yaml

```
@@ -14820,6 +14820,25 @@ plugins:
       - <<: *patch3rdParty_KPH_WACCF
         condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("WACCF CL Chillrend Patch.esp")'
 
+  - name: 'C.O.I.N.*\.esp'
```

The rest of the periods should be escaped as well:

```suggestion
  - name: 'C\.O\.I\.N.*\.esp'
```